client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -35,13 +39,35 @@ const getPageInfo = ( translate, step ) => {
 			trackerTitle: 'Account Recovery > Reset Password',
 			documentHeadTitle: translate( 'Reset Password ‹ Account Recovery' ),
 		},
-
+		[ STEPS.RESET_PASSWORD_EMAIL ]: {
+			trackerTitle: 'Account Recovery > Reset Password Email',
+			documentHeadTitle: translate( 'Reset Password < Email' ),
+		},
+		[ STEPS.RESET_PASSWORD_SMS ]: {
+			trackerTitle: 'Account Recovery > Reset Password Sms',
+			documentHeadTitle: translate( 'Reset Password ‹ SMS Message' ),
+		},
 	};
 
 	return pageInfo[ step ];
 };
 
-const getCurrentStep = ( { firstStep, isUserDataReady, isResetOptionsReady } ) => {
+const getCurrentStep = ( props ) => {
+	const {
+		firstStep,
+		isUserDataReady,
+		isResetOptionsReady,
+		selectedMethod,
+	} = props;
+
+	if ( selectedMethod ) {
+		if ( includes( selectedMethod, '_email' ) ) {
```

Comment:
Could we be more specific here and test for the two specific values which are probably "primary_email" and "secondary_email". Or if there